database/src/main/java/com/pip/chatbot/repository/food/FoodRepository.java

```diff
@@ -62,19 +66,14 @@
         return Optional.ofNullable(result.into(Mark.class));
     }
 
-    public Optional<Mark> getAvgRestaurantMark(String id) {
-        var avgMark = dsl.select(MARK_RESTAURANT.RESTAURANT_ID, avg(MARK_RESTAURANT.MARK).as("mark"))
+    public Optional<Double> getAvgRestaurantMark(int id, Double zomatoMark) {
+        var avgMark = dsl.select(avg(MARK_RESTAURANT.MARK).as("mark"))
```

Comment:
Tutaj przyjmujmy tylko id i zwracajmy wyłącznie średnią ocen userów. Logikę ich łączenia przenieśmy do serwisu, który najpierw pobierze restauracje, tak jak ona jest w bazie, potem pobierze te oceny, skleji to wszystko razem i zwróci.